CorsixTH/Lua/dialogs/fullscreen/town_map.lua

```diff
@@ -306,10 +304,8 @@ function UITownMap:draw(canvas, x, y)
       local owner_num = map.th:getPlotOwner(self.hover_plot)
       if owner_num == 0 then
         owner = _S.town_map.for_sale
-	price = "$" .. map:getParcelPrice(self.hover_plot)
-	if app.config.free_build_mode then  -- Everything is 0$ in free build mode
-	  price = "$0"
-	end
+	price = app.config.free_build_mode and "$0"  -- Everything is 0$ in free build mode
+  		or ("$" .. map:getParcelPrice(self.hover_plot
```

Comment:
This line doesn't work. It has two `(` parentheses` and zero `)` parentheses. They should be balanced.

A good way to find these kind of problems is by running the program after you changed it. It won't start at all with problems like the above. If it does run, it's a good time to check that your change is working correctly.

Except for the parentheses, the line also has a TAB and incorrect indenting.

The comment is here as well. Like above, in the middle of a statement is somewhat a bad spot.  
I don't know how you think about the comment, but it seems to me that it states a quite known fact, and thus doesn't have much value (it does say anything unexpected or special that only applies here) .

If you feel the comment has value enough to keep, I'd suggest to make a new comment above line 307 that should then cover the entire `price = ...` line rather than just the free-build bit.  
If you feel the comment isn't worth keeping, you can delete it.

Reply:
I have some issues with loading file into github from notepad++, it breaks spacing so i just copy stuff manually, and it sometimes generate that kind of problems.

About the comment - Sure, I'll delete it now